server/controllers/usersController.js

```diff
@@ -115,28 +124,35 @@ export default class Users {
     })
       .then((foundUser) => {
         if (!foundUser) {
-          return res.status(404).send({
-            message: 'This email does not exist. Sign up instead ?',
-          });
+          return res.status(404)
+            .json({
+              status: 'fail',
+              message: 'This email does not exist. Sign up instead ?',
+            });
         }
         const match = bcrypt.compareSync(req.body.password, foundUser.password);
```

Comment:
Extra variable not necessary. The expression could be passed directly to your conditional block

Reply:
Fixed